packages/client/components/NewGitHubIssueInput.tsx

```diff
@@ -131,14 +164,16 @@ const NewGitHubIssueInput = (props: Props) => {
         <Checkbox active />
         <Issue>
           <Form onSubmit={handleCreateNewIssue}>
-            <SearchInput
+            <NewIssueInput
               autoFocus
               onBlur={handleCreateNewIssue}
-              onChange={(e) => setNewIssueText(e.target.value)}
-              placeholder='New issue summary'
+              onChange={onChange}
+              name='newIssue'
+              placeholder='New issue title'
```

Comment:
What are your thoughts on using a `maxLength` here? doing so could prevent an error state from occurring, which might be nice

Reply:
I do enjoy the snarky error messages but I agree that `maxLength` is a better UX! I've pushed the update but have kept the validation check for min length